src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
```

Comment:
@buchi I was under the assumption that we have an existing @upload endpoint for regular (smaller) file uploads and a separate @tus-upload endpoint for the tus-upload. I think this is how the Guillotina API is implemented. @bloodbare @vangheem can you confirm that?